presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java

```diff
@@ -14,22 +14,18 @@
 package com.facebook.presto.hive;
 
 import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.facebook.presto.tests.FeatureSelection;
 
 import static com.facebook.presto.hive.HiveQueryRunner.createQueryRunner;
+import static com.facebook.presto.tests.TestedFeature.DELETE;
 import static io.airlift.tpch.TpchTable.getTables;
 
 public class TestHiveDistributedQueries
         extends AbstractTestDistributedQueries
 {
     public TestHiveDistributedQueries()
     {
-        super(() -> createQueryRunner(getTables()));
-    }
-
-    @Override
-    public void testDelete()
-    {
-        // Hive connector currently does not support row-by-row delete
+        super(() -> createQueryRunner(getTables()), FeatureSelection.allFeatures().excluding(DELETE));
```

Comment:
Retain the comment `// Hive connector currently does not support row-by-row delete`